src/solidity/decodeInfo.js

```diff
@@ -219,10 +230,9 @@ function getEnum (type, stateDefinitions, contractName) {
   * @param {String} typeName - name of the struct type (e.g struct <name>)
   * @param {Object} stateDefinitions  - all state definition given by the AST (including struct and enum type declaration) for all contracts
   * @param {String} contractName - contract the @args typeName belongs to
-  * @param {String} location - location of the data (storage ref| storage pointer| memory| calldata)
   * @return {Array} containing all members of the current struct type
   */
-function getStructMembers (type, stateDefinitions, contractName, location) {
```

Comment:
Please add a test for the following type: `struct X { uint a; mapping(uint=>uint) b; uint c; }`
If such a struct is used in memory, it should be identical to `struct X { uint a; uint c; }`.